pytest-selenium/pages/osweb.py

```diff
@@ -116,3 +124,30 @@ def osweb_username(self, element):
         """Get the username of the logged in user."""
         element1 = self.username(element)
         return " ".join(element1.split()[:2])
+
+    def notification_dialog_displayed(self) -> bool:
```

Comment:
Because this is a simple state on the page (bool) and not an action, this should be a `@property`.